pkg/registry/core/pod/etcd/eviction.go

```diff
@@ -116,6 +116,9 @@ func (r *EvictionREST) Create(ctx api.Context, obj runtime.Object) (runtime.Obje
 }
 
 func (r *EvictionREST) checkAndDecrement(namespace string, podName string, pdb policy.PodDisruptionBudget) (ok bool, err error) {
+	if pdb.Status.ObservedGeneration != pdb.Generation {
```

Comment:
Nothing increments Generation hence this will be always true once the controller observes a PDB for the first time.